pom.xml

```diff
@@ -141,17 +140,6 @@
                 </executions>
             </plugin>
             <plugin>
-                <groupId>org.sonatype.plugins</groupId>
```

Comment:
Are you sure you should remove this plugin? I think it is used when releasing to sonatype repo